app.js

```diff
@@ -51,10 +57,25 @@ app.post('/api/tone_chat', (req, res, next) => {
   });
 });
 
+// Endpoint to log user feedback on perceived accuracy of a customer tone
+// provided by Tone Analyzer tone_chat for a given utterance
+app.post('/log_perceived_accuracy', (req, res) => {
+  const tonesAccuracyLogEntry = req.body;
+  tonesAccuracyLogEntry.timestamp = (new Date(Date.now())).toISOString();
+  tonesAccuracyLogEntry.ip = req.ip;
```

Comment:
Move the cloudant's settings and the `/log_perceived_accuracy` endpoint to another file and call it `feedback.js`

Then you will do something like
```js
require('./feedback')(app); // Loads a feedback module 
```

In `feedback.js` you will check if cloudant parameters were provided and use cloudant otherwise, use a mock endpoint that prints out the feedback in the console.